src/main/cms/cms_menu_vtx_smartaudio.c

```diff
@@ -137,8 +135,9 @@ if (saDevice.version == 2) {
     saCmsStatusString[0] = "-FR"[saCmsOpmodel];
 
     if (saCmsFselMode == 0) {
-        saCmsStatusString[2] = "ABEFR"[saDevice.channel / 8];
-        saCmsStatusString[3] = '1' + (saDevice.channel % 8);
+        uint8_t band = saGetBand(saDevice.channel);
```

Comment:
I'm not sure I see the reason to use `int` type instead of declared return type of `saGetBand()` function — `uint8_t`. The function itself uses `uint8_t` because the `saDevice.channel` it is based on is of type `uint8_t` and it seems that there is no point to extend it up to `int`, since `uint8_t` nicely represents the set of possible values of vtx bands/channels — small and unsigned integers.

Reply:
Gcc generates considerably better code when using `int` (or `unsigned`). In general it's better to use `int` where actual size is not important. Gcc stores the value in 32bit register anyway, so no memory is wasted (unless there is big register pressure and value has to be put on stack)

The same applies for return type - `band` is integer quantity, there is no need to specify its size.

`saDevice.channel` is different - space saving is (a bit) important (stored in struct/RAM) and `channel` is transferred outside of FC, so it's type should match.